docs/pages/careers/react-engineer-docs-infra.md

```diff
@@ -0,0 +1,113 @@
+# React Engineer — Docs Infra
+
+<p class="description">You will drive the development of the documentation platform that powers all MUI products, enabling it to grow and establish itself as a mature product.</p>
+
+## Details of the role
+
+- **Location**: Remote (preference for UTC-5 to UTC+2).
+- **Type of work**: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501#494af1f358794028beb4b7697b5d3102)).
+- **Level**: [IC3 or above](https://mui-org.notion.site/Leveling-at-MUI-5c30f9bfe65149d697f346447cef9db1).
```

Comment:
I wondered if we could've gone even a bit lower than this, like IC2 or above, but given it's infrastructure-heavy, maybe parting from Senior (IC3) feels safer.

Reply:
I don't think IC2 can work for this role. I think execution can make the difference between an amazing UX and a poor one.